pkg/controller/nodeconfigpod/testdata/eks/sys/fs/cgroup/cpuset/kubepods.slice/kubepods-pod03840283_da85_4518_a860_2d1da454207c.slice/docker-5bdb9a25f236e703161902376f9377336c072a2c4fca54a5100e8e07fb7d4280.scope/cpuset.cpus

```diff
@@ -0,0 +1 @@
+0-2
```

Comment:
why are these files committed?

Reply:
these are golden files for unit tests verifying fallback of missing runtime info in CRI